snpm_cp.m

```diff
@@ -782,6 +798,45 @@ function snpm_cp(CWD)
 %-Loop over planes (breaks out after first loop if bVolm)
 %-----------------------------------------------------------------------
 nP = [];
+
+if(UseRapidPT >= 1)
+    
+    params.N = size(X,1);
+    params.V = size(X,2);
+    params.nPerm = nPerm;
+    params.xdim = xdim; 
+    params.ydim = ydim; 
+    params.zdim = zdim; 
+    params.origin = ORIGIN; 
+    params.nGroup1 = size(find(iCond == 1),2); 
+    params.nGroup2 = params.N - params.nGroup1; 
+
+    datasetInfo = strcat(num2str(params.N),'_',num2str(params.nGroup1),'_',num2str(params.nGroup2));
+    runInfo = strcat(datasetInfo,'_',num2str(nPerm),'.mat');
+    save(strcat('params',runInfo),'params');
+
+    
+    fullpath = mfilename('fullpath');
+    % We only need the directory of snpm_cp so we remove the 7 chars
+    % associated to 'snpm_cp' in fullpath.
+    RapidPT_path = strcat(fullpath(1:end-7),'RapidPT_min');
+    addpath(RapidPT_path);
+
+    write = 0;
+    [outputs, timings] = TwoSampleRapidPT(X, nPerm, params.nGroup1, write, RapidPT_path);
+    MaxT = outputs.MaxT;
+    save(strcat('timings',runInfo),'timings');
+
+    % Save variables for snpm_pp
+    MaxT = [MaxT',-MaxT'];
```

Comment:
Note that MaxT should have the *correctly* *labeled* Max and Min T stat in the first row; presently it seems like MaxT(1,1) and MaxT(1,2) are just arbitrary permutations.  
ALSO, the second column is the *negation* of the minimum T values, i.e., the MaxT(:,2) should be all positive.

Could you revise this so that MaxT is compatible with snpm_cp?